content/articles/deployment-editor.md

```diff
@@ -1,110 +0,0 @@
----
```

Comment:
What do you think of making use of https://github.com/dnsimple/dnsimple-support/blob/main/_redirects to redirect https://support.dnsimple.com/articles/deployment-editor/ to the new "Record Editor for Integrated Zones" page? (It looks like it can be used for such redirects)

Reply:
Ooo I didn't know we had this. Good idea! I'll add in the redirect.